firestore/src/ios/credentials_provider_desktop.cc

```diff
@@ -63,7 +90,7 @@ StatusOr<Token> ConvertToken(const Future<std::string>& future, App& app) {
     // is "from a different error domain".
     // TODO(b/174485290) Map `AuthError` values to Firestore `Error` values more
     // intelligently so as to enable retries when appropriate.
-    return Status(Error::kErrorUnknown,
+    return Status(ConvertAuthErrorToFirestoreError(future.error()),
```

Comment:
Please update the comment above this line.

Reply:
Done.